README.md

```diff
@@ -160,6 +160,11 @@ The map of containers consists of the name of the container mapped to the contai
 * `build` (object, optional): Parameters mapped to Docker's `build`.
 	* `context` (string)
 	* `file` (string)
+* `push` (object, optional): Parameters related to Docker's `push`.
+  * `skip` (boolean): prevents pushing image when `push` is called
```

Comment:
no groups are defined in `crane.yml`, it's just that this would centralize which containers should be pushed in one place (rather than having a toggle per container).